android/app/src/main/java/org/mozilla/magnet/notifications/ReceiverNotificationDelete.java

```diff
@@ -15,11 +19,20 @@
     @Override
     public void onReceive(Context context, Intent intent) {
         Log.d(TAG, "on receive");
+
         String action = intent.getAction();
         if (!action.startsWith("notification-dismiss")) { return; }
         int id = (int) intent.getExtras().get("id");
+
         clearNotification(context, id);
         sendBroadcast(context);
+
+        Api api = new ApiMagnet(context);
+        api.post(
+            "tracking",
+            ApiAnalytics.createEvent("system", "notification-dismiss"),
+            Api.NoopCallback
+        );
```

Comment:
If we forget about MagnetApi, this could be replaced by something like:

```java
Tracking track = new Tracking(context);
track.event("system", "notification-dismiss");
```

This exact same API could be exposed to JS.